src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import { PostInfo } from '../PostInfo';
+import { PostArr } from '../../type/post';
+import './PostList.scss';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+export const PostList: React.FC<PostArr> = ({ posts }) => (
```

Comment:
Consider renaming 'PostArr' to a more descriptive name like 'PostListProps' to better reflect its purpose.